biblio.json.bak

```diff
@@ -1,4 +1,12 @@
 {  
+  "Ceremony": {
+    "title": "Ceremony Design and Analysis",
+    "href": "https://eprint.iacr.org/2007/399.pdf",
+    "authors": ["Carl Ellison"],
+    "date": "2007"
+  },
+
+	
   "GeoJSON": {
     "title": "The GeoJSON Format Specification",
     "href": "http://geojson.org/geojson-spec.html"
```

Comment:
Also, the PDF file below should also be removed from the commit. (I can't comment on the file itself since it's binary.)